src/views/Contribute/Contribute.tsx

```diff
@@ -0,0 +1,311 @@
+
+      <Accordion allowMultiple allowToggle defaultIndex={[0]}>
+        <FAQItem question="How can I add my construct to Construct Hub?">
+          <Stack spacing={4}>
+            <Text>
+              Constructs intended for Construct Hub must be published to the{" "}
+              <FAQLink href="https://www.npmjs.com/">NPM registry</FAQLink>{" "}
+              under a permissive license (such as Apache, BSD, EPL, MPL-2.0, ISC
+              and CDDL or MIT) and annotated with a keyword recognized by
+              Construct Hub (awscdk, cdk8s, or cdktf).
+            </Text>
+
+            <Text>
+              Additionally, since one of the main goals of Construct Hub is to
+              enable an ecosystem of constructs that can be consumed by all CDK
+              languages, your library *must* be compiled with{" "}
+              <FAQLink href="https://aws.github.io/jsii/">JSII</FAQLink>, a
+              TypeScript-based tool for building multi-language libraries.
+              Construct Hub leverages the type information produced by the JSII
+              compiler to render a rich multi-language API reference for each
+              construct.
+            </Text>
+
+            <Text>
+              Construct Hub continuously monitors the NPM Registry. Packages
+              that meet the above requirements appear in Construct Hub within
+              5-10 minutes. If your package does not appear in Construct Hub,
+              but meets these requirements, please file an issue against our
+              GitHub repository.
+            </Text>
+
+            <Text>
+              The community has provided some great resources about publishing
+              construct libraries that meet Construct Hub requirements. For
+              example, see “A Beginner&apos;s Guide to Create AWS CDK Construct
+              Library with{" "}
+              <FAQLink href="https://dev.to/aws-builders/a-beginner-s-guide-to-create-aws-cdk-construct-library-with-projen-5eh4">
+                projen
+              </FAQLink>
+              ” by <FAQLink href="https://hayao-k.dev/">hayao-k</FAQLink>.
+            </Text>
+
+            <Text>
+              If you already have a library written in TypeScript and want to
+              migrate it to JSII so it can be included in Construct Hub, see the{" "}
+              <FAQLink href="https://aws.github.io/jsii/user-guides/#library-author-guide">
+                JSII library author guide.
+              </FAQLink>
+            </Text>
+          </Stack>
+        </FAQItem>
+
+        <FAQItem question="What are the best practices for improving discoverability of a construct library on Construct Hub?">
+          <Stack spacing={4}>
+            <Text>
+              To improve your construct library’s visibility on Construct Hub,
+              follow these best practices:
+            </Text>
+            <OrderedList pl={8} sx={{ li: { mt: 4 } }}>
+              <ListItem>
```

Comment:
Can we decrease the margin between the paragraph and the first list item? see screenshot:

<img width="1029" alt="Screen Shot 2021-11-17 at 2 35 17 PM" src="https://user-images.githubusercontent.com/5008987/142293571-fc10bae2-d2fe-4e68-b3a9-4d8e81ef6a0e.png">

Reply:
Oh yeah, this shouldn't have so much margin on the top lol